go/go.mod

```diff
@@ -2,13 +2,12 @@ module github.com/cucumber/gherkin/go/v26
 
 require (
 	github.com/cucumber/messages/go/v21 v21.0.1
-	github.com/cucumber/messages/go/v22 v22.0.0
```

Comment:
It looks like Renovate upgraded this incorrectly. I suspect the correct solution would be to remove v21.

https://github.com/cucumber/gherkin/commit/d4998a6a652fa97b746c3d38949f50189218ade9

Reply:
See #133 because Renovate is buggy (https://github.com/cucumber/gherkin/pull/120#issuecomment-1634352242).